examples/devmode.yaml

```diff
@@ -1,5 +1,5 @@
 system:
-    plugin_path: <specify lib root for plugins>
```

Comment:
what's the default ? using that is also an option perhaps ?

Reply:
Currently, there is no default. Perhaps there should be. This could be raised as an issue under `aeternity`.